### ISSUE TITLE
java.lang.NoSuchMethodError: No static method Text-6FffQQw

### ISSUE THREAD

**Issue by smallworldpc** [vanpra/compose-material-dialogs#84]
I get the below error after updating the compose version.

`java.lang.NoSuchMethodError: No static method Text-6FffQQw(Ljava/lang/String;Landroidx/compose/ui/Modifier;JJLandroidx/compose/ui/text/font/FontStyle;Landroidx/compose/ui/text/font/FontWeight;Landroidx/compose/ui/text/font/FontFamily;JLandroidx/compose/ui/text/style/TextDecoration;Landroidx/compose/ui/text/style/TextAlign;JLandroidx/compose/ui/text/style/TextOverflow;ZILkotlin/jvm/functions/Function1;Landroidx/compose/ui/text/TextStyle;Landroidx/compose/runtime/Composer;III)V in class Landroidx/compose/material/TextKt; or its super classes (declaration of 'androidx.compose.material.TextKt' appears in /data/app/com.example-Y03XriHO9sVbz_0gjKYDeA==/base.apk)`

**Verisons & dependencies used for this lib:**
kotlin_version = '1.5.10'
compose_version = '1.0.0-beta08' 
implementation("io.github.vanpra.compose-material-dialogs:datetime:0.4.2")
coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:1.1.5")

Before updating the kotlin and compose versions, it used to work fine. Please let me know if I'm doing anything wrong here.

**Comment by forntoh** [comment]
Update to compose `1.0.0-beta09`

**Comment by sapuglha** [comment]
Happening with Compose `1.0.0-beta09` too.

```
2021-06-18 13:25:43.269 7047-7047/com.bankgreenwood.app.dev E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.bankgreenwood.app.dev, PID: 7047
    java.lang.NoSuchMethodError: No static method Text-6FffQQw(Ljava/lang/String;Landroidx/compose/ui/Modifier;JJLandroidx/compose/ui/text/font/FontStyle;Landroidx/compose/ui/text/font/FontWeight;Landroidx/compose/ui/text/font/FontFamily;JLandroidx/compose/ui/text/style/TextDecoration;Landroidx/compose/ui/text/style/TextAlign;JLandroidx/compose/ui/text/style/TextOverflow;ZILkotlin/jvm/functions/Function1;Landroidx/compose/ui/text/TextStyle;Landroidx/compose/runtime/Composer;III)V in class Landroidx/compose/material/TextKt; or its super classes (declaration of 'androidx.compose.material.TextKt' appears in /data/app/~~cOV-E3Hf1HckUcei47mCQA==/com.bankgreenwood.app.dev-sKYR0rZXxUBcIP33Zv8GXg==/base.apk)
        at com.vanpra.composematerialdialogs.datetime.datepicker.DatePickerKt.CalendarHeader(DatePicker.kt:388)
        at com.vanpra.composematerialdialogs.datetime.datepicker.DatePickerKt.DatePickerImpl(DatePicker.kt:107)
        at com.vanpra.composematerialdialogs.datetime.datepicker.DatePickerKt.datepicker(DatePicker.kt:87)
        at com.bankgreenwood.app.ui.components.DatePickerCommonKt$DatePickerSetUp$1.invoke(DatePickerCommon.kt:36)
        at com.bankgreenwood.app.ui.components.DatePickerCommonKt$DatePickerSetUp$1.invoke(DatePickerCommon.kt:35)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at com.vanpra.composematerialdialogs.MaterialDialog$build$2$2.invoke(MaterialDialog.kt:197)
        at com.vanpra.composematerialdialogs.MaterialDialog$build$2$2.invoke(MaterialDialog.kt:195)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.material.SurfaceKt$Surface$6.invoke(Surface.kt:267)
        at androidx.compose.material.SurfaceKt$Surface$6.invoke(Surface.kt:254)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.material.SurfaceKt.Surface-F-jzlyU(Surface.kt:251)
        at androidx.compose.material.SurfaceKt.Surface-F-jzlyU(Surface.kt:110)
        at com.vanpra.composematerialdialogs.MaterialDialog$build$2.invoke(MaterialDialog.kt:185)
        at com.vanpra.composematerialdialogs.MaterialDialog$build$2.invoke(MaterialDialog.kt:179)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1$1.invoke(Utils.kt:25)
        at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1$1.invoke(Utils.kt:24)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.material.TextKt.ProvideTextStyle(Text.kt:252)
        at androidx.compose.material.MaterialThemeKt$MaterialTheme$1.invoke(MaterialTheme.kt:81)
        at androidx.compose.material.MaterialThemeKt$MaterialTheme$1.invoke(MaterialTheme.kt:80)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.material.MaterialThemeKt.MaterialTheme(MaterialTheme.kt:72)
        at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1.invoke(Utils.kt:24)
2021-06-18 13:25:43.270 7047-7047/com.bankgreenwood.app.dev E/AndroidRuntime:     at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1.invoke(Utils.kt:23)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1$2.invoke(AndroidDialog.android.kt:140)
        at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1$2.invoke(AndroidDialog.android.kt:139)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.ui.window.AndroidDialog_androidKt.DialogLayout(AndroidDialog.android.kt:386)
        at androidx.compose.ui.window.AndroidDialog_androidKt.access$DialogLayout(AndroidDialog.android.kt:1)
        at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1.invoke(AndroidDialog.android.kt:137)
        at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1.invoke(AndroidDialog.android.kt:134)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.ui.window.DialogLayout.Content(AndroidDialog.android.kt:193)
        at androidx.compose.ui.platform.AbstractComposeView$ensureCompositionCreated$1.invoke(ComposeView.android.kt:228)
        at androidx.compose.ui.platform.AbstractComposeView$ensureCompositionCreated$1.invoke(ComposeView.android.kt:227)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.ui.platform.CompositionLocalsKt.ProvideCommonCompositionLocals(CompositionLocals.kt:148)
        at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt$ProvideAndroidCompositionLocals$3.invoke(AndroidCompositionLocals.android.kt:114)
        at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt$ProvideAndroidCompositionLocals$3.invoke(AndroidCompositionLocals.android.kt:113)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt.ProvideAndroidCompositionLocals(AndroidCompositionLocals.android.kt:106)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1$1$3.invoke(Wrapper.android.kt:162)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1$1$3.invoke(Wrapper.android.kt:161)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:215)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(Wrapper.android.kt:161)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(Wrapper.android.kt:144)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
        at androidx.compose.runtime.ComposerKt.invokeComposable(Composer.kt:3324)
        at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:2575)
        at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:2571)
        at androidx.compose.runtime.SnapshotStateKt.observeDerivedStateRecalculations(SnapshotState.kt:523)
        at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:2564)
2021-06-18 13:25:43.272 7047-7047/com.bankgreenwood.app.dev E/AndroidRuntime:     at androidx.compose.runtime.ComposerImpl.composeContent$runtime_release(Composer.kt:2515)
        at androidx.compose.runtime.CompositionImpl.composeContent(Composition.kt:476)
        at androidx.compose.runtime.Recomposer.composeInitial$runtime_release(Recomposer.kt:727)
        at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:2978)
        at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:432)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(Wrapper.android.kt:144)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(Wrapper.android.kt:135)
        at androidx.compose.ui.platform.AndroidComposeView.setOnViewTreeOwnersAvailable(AndroidComposeView.android.kt:724)
        at androidx.compose.ui.platform.WrappedComposition.setContent(Wrapper.android.kt:135)
        at androidx.compose.ui.platform.WrappedComposition.onStateChanged(Wrapper.android.kt:187)
        at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:354)
        at androidx.lifecycle.LifecycleRegistry.addObserver(LifecycleRegistry.java:196)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(Wrapper.android.kt:142)
        at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(Wrapper.android.kt:135)
        at androidx.compose.ui.platform.AndroidComposeView.onAttachedToWindow(AndroidComposeView.android.kt:811)
        at android.view.View.dispatchAttachedToWindow(View.java:20479)
        at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3489)
        at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3496)
        at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3496)
        at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3496)
        at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:3496)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2417)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1952)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8171)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)
        at android.view.Choreographer.doCallbacks(Choreographer.java:796)
        at android.view.Choreographer.doFrame(Choreographer.java:731)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

```

**Comment by forntoh** [comment]
@sapuglha This library needs to be updated to compose `beta09` it is still on `beta08`

https://github.com/vanpra/compose-material-dialogs/blob/8cca13eddf614a598b0f77fa89e682f395f30ef2/buildSrc/src/main/kotlin/Dependencies.kt#L35

**Comment by vanpra** [comment]
A new release with an updated compose version will be coming out tomorrow.

**Comment by sapuglha** [comment]
Thanks!

**Comment by vanpra** [comment]
Should be fixed in release 0.4.3

**Comment by Alexey-Gvozditskiy** [comment]
Lib Version 0.5.1
Compose 1.0.0

Fatal Exception: java.lang.NoSuchMethodError: No static method Text-6FffQQw(Ljava/lang/String;Lr0/f;JJLs1/i;Ls1/k;Ls1/e;JLx1/d;Lx1/c;JLx1/j;ZILjg/l;Lo1/a0;Lg0/i;III)V in class Lb0/b2; or its super classes (declaration of 'b0.b2' appears in base.apk)
       at com.vanpra.composematerialdialogs.datetime.timepicker.TimePickerKt.TimePickerTitle-rAjV9yQ(TimePickerKt.java:908)
       at com.vanpra.composematerialdialogs.datetime.timepicker.TimePickerKt.TimePickerImpl(TimePickerKt.java:477)
       at com.vanpra.composematerialdialogs.datetime.timepicker.TimePickerKt.timepicker(TimePickerKt.java:564)
       at com.basenjiapps.breeze.settings.presentation.SettingsPageKt$SettingsPage$1.invoke(SettingsPageKt.java:114)
       at com.basenjiapps.breeze.settings.presentation.SettingsPageKt$SettingsPage$1.invoke(SettingsPageKt.java:10)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at com.vanpra.composematerialdialogs.MaterialDialog$build$2$2.invoke(MaterialDialog.java:177)
       at com.vanpra.composematerialdialogs.MaterialDialog$build$2$2.invoke(MaterialDialog.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.material.SurfaceKt$Surface$6.invoke(SurfaceKt.java:223)
       at androidx.compose.material.SurfaceKt$Surface$6.invoke(SurfaceKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.material.SurfaceKt.Surface-F-jzlyU(SurfaceKt.java:334)
       at androidx.compose.material.SurfaceKt.Surface-F-jzlyU(SurfaceKt.java:411)
       at com.vanpra.composematerialdialogs.MaterialDialog$build$2.invoke(MaterialDialog.java:147)
       at com.vanpra.composematerialdialogs.MaterialDialog$build$2.invoke(MaterialDialog.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1$1.invoke(UtilsKt.java:29)
       at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1$1.invoke(UtilsKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.material.TextKt.ProvideTextStyle(TextKt.java:92)
       at androidx.compose.material.MaterialThemeKt$MaterialTheme$1.invoke(MaterialThemeKt.java:31)
       at androidx.compose.material.MaterialThemeKt$MaterialTheme$1.invoke(MaterialThemeKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.material.MaterialThemeKt.MaterialTheme(MaterialThemeKt.java:415)
       at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1.invoke(UtilsKt.java:43)
       at com.vanpra.composematerialdialogs.UtilsKt$ThemedDialog$1.invoke(UtilsKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1$2.invoke(AndroidDialog_androidKt.java:28)
       at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1$2.invoke(AndroidDialog_androidKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.ui.window.AndroidDialog_androidKt.DialogLayout(AndroidDialog_androidKt.java:212)
       at androidx.compose.ui.window.AndroidDialog_androidKt.access$DialogLayout(AndroidDialog_androidKt.java)
       at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1.invoke(AndroidDialog_androidKt.java:44)
       at androidx.compose.ui.window.AndroidDialog_androidKt$Dialog$dialog$1$1$1.invoke(AndroidDialog_androidKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.ui.window.DialogLayout.Content(DialogLayout.java:16)
       at androidx.compose.ui.platform.AbstractComposeView$ensureCompositionCreated$1.invoke(AbstractComposeView.java:21)
       at androidx.compose.ui.platform.AbstractComposeView$ensureCompositionCreated$1.invoke(AbstractComposeView.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.ui.platform.CompositionLocalsKt.ProvideCommonCompositionLocals(CompositionLocalsKt.java:280)
       at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt$ProvideAndroidCompositionLocals$3.invoke(AndroidCompositionLocals_androidKt.java:31)
       at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt$ProvideAndroidCompositionLocals$3.invoke(AndroidCompositionLocals_androidKt.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.ui.platform.AndroidCompositionLocals_androidKt.ProvideAndroidCompositionLocals(AndroidCompositionLocals_androidKt.java:274)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1$1$3.invoke(WrappedComposition.java:27)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1$1$3.invoke(WrappedComposition.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocalKt.java:28)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(WrappedComposition.java:157)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1$1.invoke(WrappedComposition.java:8)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:48)
       at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambdaImpl.java:8)
       at androidx.compose.runtime.ComposerKt.invokeComposable(ComposerKt.java:15)
       at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(ComposerImpl.java:19)
       at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(ComposerImpl.java)
       at androidx.compose.runtime.SnapshotStateKt.observeDerivedStateRecalculations(SnapshotStateKt.java:46)
       at androidx.compose.runtime.ComposerImpl.doCompose(ComposerImpl.java:137)
       at androidx.compose.runtime.ComposerImpl.composeContent$runtime_release(ComposerImpl.java:18)
       at androidx.compose.runtime.CompositionImpl.composeContent(CompositionImpl.java:17)
       at androidx.compose.runtime.Recomposer.composeInitial$runtime_release(Recomposer.java:33)
       at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(ComposerImpl.java:16)
       at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(ComposerImpl.java:16)
       at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(ComposerImpl.java:16)
       at androidx.compose.runtime.CompositionImpl.setContent(CompositionImpl.java:15)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(WrappedComposition.java:87)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(WrappedComposition.java:2)
       at androidx.compose.ui.platform.AndroidComposeView.setOnViewTreeOwnersAvailable(AndroidComposeView.java:11)
       at androidx.compose.ui.platform.WrappedComposition.setContent(WrappedComposition.java:12)
       at androidx.compose.ui.platform.WrappedComposition.onStateChanged(WrappedComposition.java:28)
       at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:14)
       at androidx.lifecycle.LifecycleRegistry.addObserver(LifecycleRegistry.java:92)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(WrappedComposition.java:48)
       at androidx.compose.ui.platform.WrappedComposition$setContent$1.invoke(WrappedComposition.java:2)
       at androidx.compose.ui.platform.AndroidComposeView.onAttachedToWindow(AndroidComposeView.java:122)
       at android.view.View.dispatchAttachedToWindow(View.java:22010)
       at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4291)
       at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4298)
       at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4298)
       at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4298)
       at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:4298)
       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3135)
       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2618)
       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:9971)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1010)
       at android.view.Choreographer.doCallbacks(Choreographer.java:809)
       at android.view.Choreographer.doFrame(Choreographer.java:744)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:995)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:246)
       at android.app.ActivityThread.main(ActivityThread.java:8512)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1139)
